src/ruby/spec/channel_credentials_spec.rb

```diff
@@ -60,6 +60,31 @@ def load_test_certs
       blk = proc { GRPC::Core::ChannelCredentials.new(nil, '', nil) }
       expect(&blk).to raise_error
     end
+
+    it 'can be constructed with a fallback credential' do
+      blk = proc {
+        fallback = GRPC::Core::ChannelCredentials.new
+        GRPC::Core::ChannelCredentials.new(fallback)
+      }
+      expect(&blk).not_to raise_error
+    end
+
+    it 'fails gracefully constructed with a non-C-extension object' do
```

Comment:
general comment: it would be nice have a test which involves performing an RPC using a stub created with XDS credentials.

For example, it would be nice to have basically the same test as the `test_xds_creds_fallback_ssl` test in  https://github.com/grpc/grpc/pull/25365/files

Reply:
test added